player/lua/osc.lua

```diff
@@ -248,8 +248,16 @@ local state = {
     using_video_margins = false,
     border = true,
     maximized = false,
+    visibility_cycle = nil,
```

Comment:
`visibility_cycle` and `visibility_cycle_list_length` are declared in `state`, but `visibility_cycle()` uses `visibility_cycle_values` and `visibility_cycle_length`. But you can just check `#state.visibility_cycle_values` instead of storing the length.

Reply:
Doing stuff late at night.

Reply:
`visibility_cycle_values` is still undeclared.